be/src/service/doris_main.cpp

```diff
@@ -503,6 +503,7 @@ int main(int argc, char** argv) {
         // this will cause coredump for ASAN build when running regression test,
         // disable temporarily.
         doris::ExecEnv::GetInstance()->task_pool_mem_tracker_registry()->logout_task_mem_tracker();
+        doris::ExecEnv::GetInstance()->process_mem_tracker()->enable_print_log_usage();
```

Comment:
If this method has to be called and it only set a property value. I think the property value should be a config.

Reply:
`enable_print_log_usage` is reset to true every 1s, this is to avoid frequent log printing when process mem tracker exceeds mem limit.